docs/api-docs/combineReducers.md

```diff
@@ -0,0 +1,27 @@
+# `combineReducers(reducersMap)`
+
+* `reducersMap: Object<string, ReducerFunction>` &ndash; a map of keys to nested
+  reducers, just like the `combineReducers` you would find in Redux itself.
+
+## Notes
+
+Reducer composition is key to a clean Redux application. The built-in Redux
+`combineReducers` won't work for nested reducers that use `loop`, so we included
+one that is aware that some reducers might have side effects. The `combineReducers`
+in redux-loop knows how to compose cmds as well as state from nested reducers
+so that your effects tree is always separate from your state tree. It's also
+completely compatible with the one in Redux, so there should be no issues
+switching to this implementation.
+
+## Examples
+
+```js
+import { combineReducers } from 'redux-loop';
+import reducerWithSideEffects from './reducer-with-side-effects';
+import plainReducer from './plain-reducer';
+
+export default combineReducers({
+  withEffects: reducerWithSideEffects,
+  plain: plainReducer
+});
```

Comment:
I think it'd be good to mention the immutable/other helper libraries here too.

Reply:
I'll take a stab at this, but I didn't use the immutable libraries myself, so I'm not completely certain how they're used.

Reply:
sorry I should have given more detail. Maybe change it to something like this.

### Using Immutable.js

If you're using
[immutable.js](https://github.com/facebook/immutable-js/) with your reducer,
you'll want to use the ImmutableJS version of combineReducers from the [redux-loop-immutable](https://github.com/redux-loop/redux-loop-immutable/) package. It has the same API as the non-immutable version except that the state it returns is an Immutable `Map` instead of an `Object`.

```js
import { combineReducers } from 'redux-loop-immutable';
import reducerWithSideEffects from './reducer-with-side-effects';
import plainReducer from './plain-reducer';

export default combineReducers({
  withEffects: reducerWithSideEffects,
  plain: plainReducer
});
```